gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/ser/Serializers.java

```diff
@@ -54,6 +55,8 @@ public MessageSerializer<?> simpleInstance() {
                 return new GraphSONMessageSerializerV3d0();
             case SerTokens.MIME_GRAPHSON_V1D0:
                 return new GraphSONMessageSerializerGremlinV1d0();
+            case SerTokens.MIME_GRAPHSON_V1D0_SPARSE:
```

Comment:
What serializer will be used for `MIME_JSON`?

Reply:
On server side it depends on the order of serializers defined in serializerSettings. Which is the first in the serializerSettings field will be picked for application/json.

I believe this snippet of code is used in gremlin driver. Not sure how it works on client side.

Reply:
Correction. Seems like on client side `GraphSONMessageSerializerV3d0` will be used as serializer for `MIME_JSON` as you can see on L54(old) in the same class.